pyof/foundation/network_types.py

```diff
@@ -332,7 +341,7 @@ def pack(self, value=None):
                                                        type(self).__name__)
             raise PackException(msg)
 
-    def unpack(self, buffer, offset=0):
+    def unpack(self, buff, offset=0):
```

Comment:
Why?

Reply:
This method in super class has the name buff not buffer.